MetaMask/metamask-mobile
# [FIX] Update token list when wallet change

**Description**
Token detection not working when fresh install the app and import a fresh account by private key

**Proposed Solution**
Fetch the token list when the address changes

**Test Cases**
Case1:
* Fresh install
* Import wallet with SRP
* wallet with tokens detected
* import wallet via Private key
* Expected tokens be detected as well
* import tokens on both wallet

**Screenshots/Recordings**
IOS: https://recordit.co/yS8vK1Bffh
Android: https://recordit.co/hB5SdcSLs0

_If applicable, add screenshots and/or recordings to visualize the before and after of your change_

**Issue**

Progresses #https://github.com/MetaMask/mobile-planning/issues/614

**Checklist**
